Crashing on "transitions is not defined"

Our deployment of rubbernecker has been crashing with the following entries in the log:
```
2017-08-08T15:45:06.50+0100 [APP/PROC/WEB/0] OUT GET / 200 1753.492 ms - 1063
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR /home/vcap/app/lib/storyFetcher.js:103
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR                     callback("Could not load the transitions for the story: " + story.id, transitions);
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR                                                                                           ^
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR ReferenceError: transitions is not defined
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at Request.getStories [as _callback] (/home/vcap/app/lib/storyFetcher.js:103:91)
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at Request.self.callback (/home/vcap/app/node_modules/request/request.js:186:22)
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at emitTwo (events.js:125:13)
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at Request.emit (events.js:213:7)
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at Request.<anonymous> (/home/vcap/app/node_modules/request/request.js:1060:10)
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at emitOne (events.js:115:13)
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at Request.emit (events.js:210:7)
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at IncomingMessage.<anonymous> (/home/vcap/app/node_modules/request/request.js:980:12)
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at Object.onceWrapper (events.js:314:30)
2017-08-08T15:45:06.54+0100 [APP/PROC/WEB/0] ERR     at emitNone (events.js:110:20)
2017-08-08T15:45:06.55+0100 [APP/PROC/WEB/0] ERR npm ERR! code ELIFECYCLE
2017-08-08T15:45:06.55+0100 [APP/PROC/WEB/0] ERR npm ERR! errno 1
2017-08-08T15:45:06.55+0100 [APP/PROC/WEB/0] ERR npm ERR! rubbernecker@0.1.0 start: `node ./bin/www`
2017-08-08T15:45:06.55+0100 [APP/PROC/WEB/0] ERR npm ERR! Exit status 1
2017-08-08T15:45:06.55+0100 [APP/PROC/WEB/0] ERR npm ERR!
2017-08-08T15:45:06.55+0100 [APP/PROC/WEB/0] ERR npm ERR! Failed at the rubbernecker@0.1.0 start script.
2017-08-08T15:45:06.55+0100 [APP/PROC/WEB/0] ERR npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2017-08-08T15:45:06.55+0100 [APP/PROC/WEB/0] ERR npm ERR! A complete log of this run can be found in:
2017-08-08T15:45:06.55+0100 [APP/PROC/WEB/0] ERR npm ERR!     /home/vcap/app/.npm/_logs/2017-08-08T14_45_06_554Z-debug.log
2017-08-08T15:45:06.56+0100 [APP/PROC/WEB/0] OUT Exit status 1
```

--- a/lib/storyFetcher.js
+++ b/lib/storyFetcher.js
@@ -100,7 +100,7 @@ internals.getStoryTransitions = function(res, cb) {
                 if (!error && response.statusCode == 200) {
                     callback(null, JSON.parse(body));
                 } else {
-                    callback("Could not load the transitions for the story: " + story.id, transitions);
+                    callback("Could not load the transitions for the story: " + story.id + " due to: " + response.statusCode, null);
                 }
             });
         });

